### PR TITLE
Recheck and fix int64_t and uint64_t print formats

### PR DESCRIPTION
**Pull request by efdeb**
Just rechecked int64_t and uint64_t print formats in all code, and fixed some of them.
Have no more compilation warning about them on Ubuntu 20.04 LTS 64 bits.

### CODE REVIEW

File: hardware/EvohomeWeb.cpp [hunk 1]
```diff
@@ -709,8 +709,9 @@ void CEvohomeWeb::DecodeDHWState(temperatureControlSystem* tcs)
 		}
 		else if ((result[0][1] != szId) || (result[0][2] != ndevname))
 		{
+			uint64_t DevRowIdx = (uint64_t) std::stoull(result[0][0]);
```
Comment (gizmocuz) [inline hardware/EvohomeWeb.cpp]: I don't think you need to use (uint64_t) here as it is already a uint64_t

Comment (efdeb) [reply hardware/EvohomeWeb.cpp]: In fact, std::chrono::milliseconds is a int64_t, not a uint64_t. That's the reason why i converted. Anyway, this is not strictly needed as the conversion will be implicit when assigning the value to the uint64_t variable.

Up to you : just tell me.

Comment (gizmocuz) [reply hardware/EvohomeWeb.cpp]: True, but std::stoull is already returning a uint64_t, so casting to this seems double (compiler will fix this, but doesn't look that pretty)

Comment (efdeb) [reply hardware/EvohomeWeb.cpp]: Sorry, read too fast : I thought your comment concerned another fix.
I remove the cast, although std::stoull returns **unsigned long long** and not exactly **uint64_t**.

**Comment by gizmocuz** [comment]
Thank you again for this PR and time!!

**Pull request merged**